data/hai/hai fleets.txt

```diff
@@ -556,6 +636,53 @@ fleet "Large Hai Merchant (Human)"
 		"Headhunter (Hai)" 2
 
 
+fleet "Unfettered Logistics"
+	government "Hai (Unfettered)"
+	names "hai"
+	cargo 2
+	personality
+		confusion 25
+		timid frugal appeasing
+	variant 3
+		"Aphid" 2
+		"Lightning Bug (Pulse)"
```

Comment:
The new ships are not available in Unfettered space. Ergo, no.
However, some of the merchant sympathisers will have them when they visit.

Reply:
Wait what the Unfettered wouldn't have some of these old new ships? I thought they would instead of having the centipede and such, they do need to move crew around

Reply:
So the Scarab is an Imo Loo Mer product - which the Uhai already stole the Lightning Bug from.
And the Cicada is a Yeer e Ki product - who are possibly the most invested in the regular Hai ideology.

Both are incentivised to find ways to reclaim, through espionage or bribery, any of their ships that would aid the UHai.

The unfettered probably did have some originally. They've probably been whittled down over the decades until the Uhai are forced to rely on the services of sympathisers for some things. (Sympathisers who can do things, but not sell their ships, because that would bring down the hammer on them.)